src/Bridge/Doctrine/Orm/Filter/AbstractFilter.php

```diff
@@ -66,15 +71,8 @@ public function apply(QueryBuilder $queryBuilder, QueryNameGeneratorInterface $q
 
     /**
      * Passes a property through the filter.
-     *
-     * @param string                      $property
-     * @param mixed                       $value
-     * @param QueryBuilder                $queryBuilder
-     * @param QueryNameGeneratorInterface $queryNameGenerator
-     * @param string                      $resourceClass
-     * @param string|null                 $operationName
      */
-    abstract protected function filterProperty(string $property, $value, QueryBuilder $queryBuilder, QueryNameGeneratorInterface $queryNameGenerator, string $resourceClass, string $operationName = null);
+    abstract protected function filterProperty(string $property, $value, QueryBuilder $queryBuilder, QueryNameGeneratorInterface $queryNameGenerator, string $resourceClass, string $operationName = null/*, array $context = []*/);
```

Comment:
@dunglas The commented out `$context` in `apply` and `filterProperty` is a bit confusing. I see that `AbstractContextAwareFilter` extends from this class and overrides `apply` introducing `array $context = []`,  and then calls `$this->filterProperty(..., $context);`.
How about filterProperty, should it also be overridden in `AbstractContextAwareFilter` with adding `array $context = []`?

Reply:
oh good catch I think it should indeed

Reply:
As @meyerbaptiste reminds us, we need PHP 7.2 for this: https://3v4l.org/ktgkR